pkg/agent/controller/agent.go

```diff
@@ -410,20 +408,14 @@ func (a *Controller) updateExportedServiceStatus(name, namespace string, status
 			Message:            &msg,
 		}
 
-		numCond := len(toUpdate.Status.Conditions)
-		if numCond > 0 && serviceExportConditionEqual(&toUpdate.Status.Conditions[numCond-1], &exportCondition) {
+		// TODO: Currently we only check for conditionType Valid. Revisit this when Conflict is supported.
```

Comment:
To preserve the equivalent of the historical trace, we could add other condition types like "SyncedToBroker". But we can look into that later.

Reply:
We should, but we will have to propose these to MCS sig first.

Reply:
I wouldn't think so. The MCS API defines some common, general ones but I'd think an implementation should be able to define their own specific types - the MCS spec doesn't preclude that. Eg, the type "SyncedToBroker" would be specific to our implementation.